website/content/docs/post-processors/manifest.mdx

```diff
@@ -201,7 +201,7 @@ build.
 
 #!/bin/bash
 
-AMI_ID=$(jq -r '.builds[-1].artifact_id' manifest.json | cut -d ":" -f2)
+AMI_ID=$(jq -r '.builds[-1].artifact_id | split(": ") | .[1]' manifest.json)
```

Comment:
I believe the space after the `:` is a typo? The resulting artifact ID should be in the `<region>:<id>` format, without space, unless I'm missing something, please correct me if so.